app/channels.go

```diff
@@ -100,8 +103,9 @@ func NewChannels(s *Server, services map[ServiceKey]any) (*Channels, error) {
 		srv:             s,
 		imageProxy:      imageproxy.MakeImageProxy(s.platform, s.httpService, s.Log()),
 		uploadLockMap:   map[string]bool{},
-		collectionTypes: map[string]string{},
+		collectionTypes: map[string][]string{},
 		topicTypes:      map[string][]string{},
+		collectionMut:   sync.Mutex{},
```

Comment:
No need to initialize the mutex -- as per docs, `// The zero value for a Mutex is an unlocked mutex.` What are the tradeoffs between using a mutex vs. a https://pkg.go.dev/sync#Map?